docs/source/index.mdx

```diff
@@ -150,8 +150,8 @@ Some platforms have specific runtime requirements:
 At build time, it requires:
 
 * Gradle 8.0+
-* Kotlin 1.9+ for JVM projects
-* Kotlin 2.0+ for native, JS, and Wasm projects
+* Kotlin 2.0+ for JVM projects
```

Comment:
This is because okio pulls kotlin-stdlib 2.1. It's technically not a binary breaking change as the runtime stdlib will be compatible.